refinery/tool_manager/utils.py

```diff
@@ -31,7 +33,13 @@
 )
 # Allow JSON Schema to find the JSON pointers we define in our schemas
 JSON_SCHEMA_FILE_RESOLVER = RefResolver(
-    "{}{}{}".format('file://', os.path.abspath("tool_manager/schemas"), '/'),
+    "{}{}{}".format(
+        'file://',
+        os.path.join(
+            settings.BASE_DIR,
+            "refinery/tool_manager/schemas"
+        ),
+        '/'),
```

Comment:
Not worth worrying about, but the static strings could go in the template, and that would make the intention more clear at the get-go: `'file://{}/''.format(...)`